OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/BarChartScaling.cs

```diff
@@ -0,0 +1,60 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using System.Collections.Generic;
+
+namespace OpenXMLSDK.Engine.Word.ReportEngine.Models.Charts
+{
+    /// <summary>
+    /// Class representating the scaling of a bar chart axis
+    /// </summary>
+    public class BarChartScaling
+    {
+        /// <summary>
+        /// Orientation of the axis
+        /// </summary>
+        public EnumValue<OrientationValues> Orientation { get; set; } = new EnumValue<OrientationValues>(OrientationValues.MinMax);
+
+        /// <summary>
+        /// Minimum axis value
+        /// </summary>
+        public double? MinAxisValue { get; set; }
+
+        /// <summary>
+        /// Maximum axis value
+        /// </summary>
+        public double? MaxAxisValue { get; set; }
+
+        /// <summary>
+        /// Construct a DocumentFormat.OpenXml.Drawing.Charts.Scaling object
+        /// </summary>
+        /// <returns></returns>
+        public Scaling GetScaling()
```

Comment:
Please move it in Render extension.
The model doesn't include logic

Reply:
Done